Levi/employee.js

```diff
@@ -0,0 +1,22 @@
+
+const createEmployee = (name, role, salary) => {
+    return {name, role, salary}
+}
+
+let micheal = createEmployee("micheal", "frontend dev", 2000)
+
+const getSalary = (employee) => {
+    return `The salary of ${employee.name} is ${employee.salary} per day`
+}
+
+console.log(getSalary(micheal))
+
+//write a function that updates the salary of the employee
+
+const updateSalary = (employee) => {
+    employee.salary = 7000
```

Comment:
Please test this, it wont work, you hard code the amount into the function and because of that the employee salary cant be changed to anything else besides 7000, pls fix that